qiling/os/windows/dlls/ntoskrnl.py

```diff
@@ -179,7 +179,7 @@ def ntoskrnl_IoCreateDevice(ql, address, params):
     device_object.CurrentIrp.value = 0
     device_object.Timer.value = 0
     device_object.Flags = 0x00000080  # DO_DEVICE_INITIALIZING
-    if params['Exclusive']:
+    if 'Exclusive' in params and params['Exclusive']:
```

Comment:
Exclusive was not passed in params when this was called from `IoCreateDevice`

Reply:
Can't this be `params.get('Exclusive') is not None`?

Reply:
That will not handle the case of Exclusive being `False` correctly